google-cloud-spanner/lib/google/cloud/spanner/backup.rb

```diff
@@ -160,6 +160,14 @@ def expire_time= time
           raise error
         end
 
+        ##
+        # The timestamp when a consistent copy of the database for the backup was
+        # taken. The version time has microseconds granularity.
+        # @return [Time]
+        def version_time
+          Convert.timestamp_to_time @grpc.version_time unless @grpc.version_time.nil?
```

Comment:
Here no need to check `nil?` value. `Convert` is already checking it. Just use 
`Convert.timestamp_to_time @grpc.version_time`

Ref:
https://github.com/googleapis/google-cloud-ruby/blob/d1aefdee2045edb5ba277d7e46329e3110b41a89/google-cloud-spanner/lib/google/cloud/spanner/convert.rb#L266-L270

Reply:
Thanks for spotting this, fixed.